scripts/import_cldr.py

```diff
@@ -691,12 +703,25 @@ def parse_calendar_datetime_skeletons(data, calendar):
                     )
 
 
-def parse_number_symbols(data, tree):
+def parse_number_symbols(data, tree, numbering_systems):
     number_symbols = data.setdefault('number_symbols', {})
-    for elem in tree.findall('.//numbers/symbols/*'):
-        if _should_skip_elem(elem):
+    default_number_system_node = tree.find('.//numbers/defaultNumberingSystem')
+
+    # If the default_number_system_node is non-existent, the package will inherit it from its parent
+    if default_number_system_node is not None:
+        number_symbols['defaultNumberingSystem'] = numbering_systems[default_number_system_node.text]
+
+    for elem in tree.findall('.//numbers/symbols'):
```

Comment:
Now that we start to have support for non-latn numbering systems, I think we should also import `<symbols>` for all the numbering systems declared in the XML – for instance, ur.xml has `<symbols>` for both `latn` and `arabext`, zh.xml has four of them... This code seems like it will erroneously merge the various numbering systems?

The resultant shape of `number_symbols` should then be something like
```json
"number_symbols": {
  "latn": {"decimal": ...},
  "arabext": {"decimal": ...},
}
```